src/Form/OptionsApi.php

```diff
@@ -103,51 +147,94 @@ public function options(): array
         return $options;
     }
 
+    /**
+     * @param array $data
+     * @return self
+     */
     protected function setData(array $data)
     {
         $this->data = $data;
         return $this;
     }
 
+    /**
+     * @param string|null $fetch
+     * @return self
+     */
     protected function setFetch(string $fetch = null)
     {
         $this->fetch = $fetch;
         return $this;
     }
 
+    /**
+     * @param $options
+     * @return self
+     */
+    protected function setOptions($options = null)
+    {
+        $this->options = $options;
+        return $this;
+    }
```

Comment:
Your attention! Added new method. Because of following line:
https://github.com/getkirby/kirby/blob/3.4.3/src/Form/OptionsApi.php#L58-L60

`$this->options` are not set anywhere. I did the same for the `OptionsQuery` class.